### PR TITLE
add known issues for global hub

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -319,4 +319,7 @@ For common Troubleshooting issues, see [Troubleshooting](troubleshooting.md).
...
-3. In `Global Hub - Policy Group Compliancy Overview` dashboards, we can check one data point by click the link of `View Offending Policies` for standard group: xxx, but after we click this link to go to the offending page, the standard group filter can not pass to the new page. The `Global Hub - Cluster Group Compliancy Overview` has the same issue.
\ No newline at end of file
+3. In `Global Hub - Policy Group Compliancy Overview` dashboards, we can check one data point by click the link of `View Offending Policies` for standard group: xxx, but after we click this link to go to the offending page, the standard group filter can not pass to the new page. The `Global Hub - Cluster Group Compliancy Overview` has the same issue.
+4. When reimport a managed hub cluster with different name. The old managed hub data will be null in the following dashboards. `Global Hub - Ad Hoc Policy Investigation` `Global Hub - What's Changed / Clusters` and `Global Hub - What's Changed / Policies`.
+5. When a managed hub is detached, all policies and applications created on managed hub will be deleted.
```
Comment (hchenxa) [inline doc/README.md:324]: the issue only exists when the managed hub is ACM 2.7 or ACM 2.8

File: doc/README.md [hunk 2]
```diff
@@ -319,4 +319,7 @@ For common Troubleshooting issues, see [Troubleshooting](troubleshooting.md).
...
+6. If managed hub running on Openshift 3.11, It can not redirect to the `RHACM Observability` page in `Global Hub - Overview` dashboard.
```
Comment (hchenxa) [inline doc/README.md:325]: it was not the managed hub running on OpenShift 3.11. it should be the managed hub have OpenShift 3.11 cluster imported and have policies running on it.

**Comment by ldpliu** [comment]
Provide draft [here](https://github.com/stolostron/rhacm-docs/pull/5729#issuecomment-1791986925), do not need to merge to main branch.

**Pull request closed without merging**